x-pack/plugins/observability/public/pages/alerts/alerts_table_t_grid.tsx

```diff
@@ -382,11 +372,12 @@ export function AlertsTableTGrid(props: AlertsTableTGridProps) {
     }
   }, []);
 
-  const leadingControlColumns = useMemo(() => {
+  const leadingControlColumns: ControlColumnProps[] = useMemo(() => {
```

Comment:
@fkanout Do we need this? Was CI complaining? Or did you add this as part of your attempt to hide the checkboxes?

Reply:
No, the CI wasn't complying. But it's a missing type even though it was part of the checkboxes fix. I would suggest keeping it.